### DISCUSSION TITLE
[FEATURE] 最近更新可以调整为最新发布的在最上方？

### DISCUSSION THREAD

**Discussion by hiifong** [hugo-fixit/FixIt#82] [Ideas]

**Comment by Lruihao** [comment]
标题的意思是想最近更新和最新发布的上下位置可以选择吗？

**Comment by hiifong** [comment]
> 标题的意思是想最近更新和最新发布的上下位置可以选择吗？

是的，按照发布时间的最新依次递减排序最近更新

**Comment by Lruihao** [comment]
> > 标题的意思是想最近更新和最新发布的上下位置可以选择吗？
> 
> 
> 
> 是的，按照发布时间的最新依次递减排序最近更新

OK，那可以添加一个选项

```yaml
    [params.section.recentlyUpdated]
      # ...
      position = top # top/bottom
```

---

只不过得过段时间才能处理了，因为我住的酒店突然被隔离了 =.=||

**Comment by hiifong** [comment]
> 
注意安全！辛苦了

**Comment by hiifong** [comment]
> > > 标题的意思是想最近更新和最新发布的上下位置可以选择吗？
> > 
> > 
> > 是的，按照发布时间的最新依次递减排序最近更新
> 
> OK，那可以添加一个选项
> 
> ```yaml
>     [params.section.recentlyUpdated]
>       # ...
>       position = top # top/bottom
> ```
> 
> 只不过得过段时间才能处理了，因为我住的酒店突然被隔离了 =.=||

现在日期显示也有点问题，最近更新的文章显示的日期都一样的

**Comment by Lruihao** [comment]
> 现在日期显示也有点问题，最近更新的文章显示的日期都一样的

- 最近更新按更新日期
- 最新发布按发布日期

**Comment by hiifong** [comment]
> > > > 标题的意思是想最近更新和最新发布的上下位置可以选择吗？
> > > 
> > > 
> > > 是的，按照发布时间的最新依次递减排序最近更新
> > 
> > 
> > OK，那可以添加一个选项
> > ```yaml
> >     [params.section.recentlyUpdated]
> >       # ...
> >       position = top # top/bottom
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 只不过得过段时间才能处理了，因为我住的酒店突然被隔离了 =.=||
> 
> 现在日期显示也有点问题，最近更新的文章显示的日期都一样的

![image](https://user-images.githubusercontent.com/89133723/158056045-1d66da55-d1e6-46db-b566-c87439245d91.png)

**Comment by Lruihao** [comment]
@hiifong 突然想到，添加到底部不合适，因为还有分页的问题

**Reply by hiifong** [reply]
> @hiifong 突然想到，添加到底部不合适，因为还有分页的问题

我的意思不是添加到底部，是按照发布时间降序来排序

**Reply by Lruihao** [reply]
> > @hiifong 突然想到，添加到底部不合适，因为还有分页的问题
> 
> 
> 
> 我的意思不是添加到底部，是按照发布时间降序来排序

按照发布时间来排序的话，其实和原本已有的时间线没有差，也无法体现出“最近更新”这个点

但是如果要体现出更新和发布时间的区别，我觉得在最近更新的时间前面加上 “更新于”（支持i18n）字样会不会更好？这样做也能消除同一篇文章时间不一样的疑惑

**Reply by hiifong** [reply]
> > > @hiifong 突然想到，添加到底部不合适，因为还有分页的问题
> > 
> > 
> > 我的意思不是添加到底部，是按照发布时间降序来排序
> 
> 按照发布时间来排序的话，其实和原本已有的时间线没有差，也无法体现出“最近更新”这个点
> 
> 但是如果要体现出更新和发布时间的区别，我觉得在最近更新的时间前面加上 “更新于”（支持i18n）字样会不会更好？这样做也能消除同一篇文章时间不一样的疑惑

确实

**Reply by hiifong** [reply]
> > > > @hiifong 突然想到，添加到底部不合适，因为还有分页的问题
> > > 
> > > 
> > > 我的意思不是添加到底部，是按照发布时间降序来排序
> > 
> > 
> > 按照发布时间来排序的话，其实和原本已有的时间线没有差，也无法体现出“最近更新”这个点
> > 但是如果要体现出更新和发布时间的区别，我觉得在最近更新的时间前面加上 “更新于”（支持i18n）字样会不会更好？这样做也能消除同一篇文章时间不一样的疑惑
> 
> 确实

有没有计划移植别的主题https://butterfly.js.org/

**Reply by Lruihao** [reply]
> > > > > @hiifong 突然想到，添加到底部不合适，因为还有分页的问题
> 
> > > > 
> 
> > > > 
> 
> > > > 我的意思不是添加到底部，是按照发布时间降序来排序
> 
> > > 
> 
> > > 
> 
> > > 按照发布时间来排序的话，其实和原本已有的时间线没有差，也无法体现出“最近更新”这个点
> 
> > > 但是如果要体现出更新和发布时间的区别，我觉得在最近更新的时间前面加上 “更新于”（支持i18n）字样会不会更好？这样做也能消除同一篇文章时间不一样的疑惑
> 
> > 
> 
> > 确实
> 
> 
> 
> 有没有计划移植别的主题https://butterfly.js.org/

工作和 free code 时间分配不过来，暂不考虑开发或移植其他主题了

**Comment by Lruihao** [comment]
https://github.com/Lruihao/FixIt/commit/c07f3a63c26c743eeb19d3082934fa9db8c8eca3
